docs/Cluster Guides/Appendix_A:_FAQ.md

```diff
@@ -0,0 +1,36 @@
+# Appendix A: FAQ
+
+### **What should I do if I need help?**
+
+Above all, the Research Computing team is here to help YOU, the researcher, make advances in your field. There are multiple decades worth of programming, system administration, scientific computing, and other technical experience between everyone on the RC team, so don’t be afraid to reach out and tap into this. Calling, emailing, and inperson vists can all be utilized and info of the following can be found at the bottom of this page. 
+
+### **How can I test my code?**
+
+Use `dev-session`, `gpu-session`, and/or `debug-session` for code testing/compiling/optimizing. This refers to 1 and 2; avoid accidentally running parallel code or a parallel compile on the head node by using the sessions we’ve built for this purpose. You should test code before running it, especially if your job is running with MPI across multiple nodes. If your code is faulty and doesn’t utilize MPI correctly (by not being configured to use multiple nodes right), then your job can “use” nodes while actually keeping them idle, wasting the resources by keeping other users from utilizing them.
```

Comment:
```suggestion
Use `dev-session`, `gpu-session`, and/or `debug-session` for code testing/compiling/optimizing. As the login node is a shared resource, we use automatic monitoring to kill processes which would slow down the login node for all users. To avoid your parallel testing or compilation processes accidentally being killed, please use the sessions we’ve built for this purpose. You should test code before running it, especially if your job is running with MPI across multiple nodes. If your code is faulty and doesn’t utilize MPI correctly (by not being correctly configured to use multiple nodes), then your job can “use” nodes while actually keeping them idle, wasting the resources by keeping other users from utilizing them.
```